test/testdata/resolver/generic_type_false_error.rb

```diff
@@ -0,0 +1,15 @@
+# typed: false
+
+extend T::Sig
+
+sig {returns(T::Array[x: Integer])}
+#                     ^^^^^^^^^^ error: Keyword arguments given to `Array`
+def example
+  []
+end
+
+sig {returns(T::Hash[y: String, z: Integer])} 
+#                    ^^^^^^^^^^^^^^^^^^^^^ error: Keyword arguments given to `Hash`
+def regression
+  {}
+end
```

Comment:
You might want to include something like

```ruby
T::Array[Integer, x: Integer].new
```

to show that we don't report the type syntax error for expressions in `# typed: false` files.